backend/app/jobs/translations/translate_project_job.rb

```diff
@@ -0,0 +1,16 @@
+module Translations
+  class TranslateProjectJob < ApplicationJob
+    queue_as :default
+
+    rescue_from(ActiveRecord::RecordNotFound) do |exception|
+      Rails.logger.error(exception)
+      Rails.logger.error(Rails.backtrace_cleaner.clean(exception.backtrace).join("\n"))
+      # TODO: handle exception tracking
+    end
+
+    def perform(project_id)
```

Comment:
I am just wondering, AR models should come with its own serialisers, therefore you can ` Translations::TranslateProjectJob.perform_later(project)` and here can be `perform(project)` <-- no need to pass `id` and then have `project = Project.find(project_id)` everywhere